vm/systemSmartContracts/proto/governance.proto

```diff
@@ -7,19 +7,29 @@ option (gogoproto.stable_marshaler_all) = true;
 
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 
+
+enum VoteType {
+    Validator = 0;
+    Account   = 1;
+}
+enum VoteValueType {
+    Yes  = 0;
+    No   = 1;
+    Veto = 2;
+}
+
 message GeneralProposal {
     bytes  IssuerAddress  = 1 [(gogoproto.jsontag) = "IssuerAddress"];
     bytes  GitHubCommit   = 2 [(gogoproto.jsontag) = "GitHubCommit"];
     uint64 StartVoteNonce = 3 [(gogoproto.jsontag) = "StartVoteNonce"];
     uint64 EndVoteNonce   = 4 [(gogoproto.jsontag) = "EndVoteNonce"];
-    int32  Yes            = 5 [(gogoproto.jsontag) = "Yes"];
-    int32  No             = 6 [(gogoproto.jsontag) = "No"];
-    int32  Veto           = 7 [(gogoproto.jsontag) = "Veto"];
-    int32  DontCare       = 8 [(gogoproto.jsontag) = "DontCare"];
-    bool   Voted          = 9 [(gogoproto.jsontag) = "Voted"];
-    repeated bytes Voters = 10 [(gogoproto.jsontag) = "Voters"];
-    bytes  TopReference   = 11 [(gogoproto.jsontag) = "TopReference"];
-    bool   Closed         = 12 [(gogoproto.jsontag) = "Closed"];
+    bytes  Yes            = 5 [(gogoproto.jsontag) = "Yes", (gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
```

Comment:
The number of votes does not tell us anything useful in quadratic voting, that's why I used vote power here also. This will keep the total voting power for each option so we don't have to recompute it when closing the proposal.

Reply:
I know we are using the voting power now but for a better analysis I think we should have also the number of votes cast for each proposal.